### DISCUSSION TITLE
Feature request: "Diagonal" seam

### DISCUSSION THREAD

**Discussion by banderies** [supermerill/SuperSlicer#3302] [General]
First of all, thank you for your incredible work on this project, this slicer is better than everything else I have tried.

I was wondering if the ability to place the seam on the back right or back left corner could be added. I prefer the "Rear" seam placement because it is the most predictable, but it would usually be better for the seam to be on the back right or left corner of my part, so I end up rotating everything on my build plate 45 degrees so that "Rear" is on the back corners of my parts. This works, but wastes a lot of space when most of my parts are rectangular. I would really appreciate it if the seam location could be placed at 45 degrees, or something similar. 

For context, one of the most popular print profiles for the Voron 2.4 automatically places everything at 45 degrees for both seam placement and surface finish (see https://github.com/AndrewEllis93/Print-Tuning-Guide). I feel like adding the 45 degree seam would be appreciated by a really large group of users.

Thanks!

**Comment by neophyl** [comment]
Use a seam sphere.  Right click the model and select add seam position.  Resize the sphere and move it wherever you want.  It acts a bit like a magnet so the seam will on the perimeter that is closest to the sphere(s).  Yes you can add multiples, however that can get confusing.  In effect with a sphere its like defining where the rear is.  You can place it inside the model too.  Have a go and you will see how it works.

Its not global so you have to add one to each part you want it to effect.

btw feature requests go in the issues section rather than discussions.

**Reply by banderies** [reply]
Thanks for the prompt reply and sorry about posting in the wrong section.

I have tried the paint seam tool with all of the seam options (rear, aligned, etc.) and it is pretty much impossible to get seams as straight as the rear seam option on it's own. It seems to just apply the global seam settings to a smaller area (painted) with unpredictable results. Better than nothing, but never as good as just rotating the parts 45 degrees and using rear seams, so that's why I was hoping corner seams could be implemented globally. Any chance you could point me to the code that is executed for rear seams? Might try to add the feature myself.

Thanks

**Comment by neophyl** [comment]
Where did I mention seam painting ?  Have you tried adding a seam sphere modifier ?

**Reply by banderies** [reply]
Oh sorry, I see now, never realized that was an option. It seems to work really well, thanks!
